.github/workflows/style.yaml

```diff
@@ -85,3 +92,49 @@ jobs:
             quit(status = length(problems))
           }
         shell: Rscript {0}
+        continue-on-error: true
+
+      - name: Styler check summary 🅾
+        run: |
+          cd ${{ github.event.repository.name }}
+          git status -s
+          if [[ -n `git status -s | grep .R` ]]
+          then {
+            echo "🙈 Code style is not correct!"
+            echo "🔀 The following differences were noted:"
+            git diff
+            if [ "${{ inputs.update }}" == "true" ]
+            then {
+              echo "Overwriting source code in new commit"
+              git config --global user.name 'Insights Engineering Bot'
+              git config --global user.email '68416928+insights-engineering-bot@users.noreply.github.com'
+              echo "Adding to git changes"
+              git status -s | grep -E ".R$|.Rmd$|.Rnw$|.Rmarkdown$" | cut -f3 -d ' '  | xargs git add
+              git commit -am "code style - update code"
+              git status
+              git log --oneline -n5
+              printenv
+              git show
+              cat $(git rev-parse --show-toplevel)/.git/config
+              git remote -v
+              echo "Checkout ${GITHUB_HEAD_REF} branch"
+              git checkout ${GITHUB_HEAD_REF}
+              echo "Pulling from ${GITHUB_HEAD_REF} branch"
+              git pull ${GITHUB_HEAD_REF}
+              echo "Pushing to ${GITHUB_HEAD_REF}"
+              git push -v
+              echo "Pushed"
+            }
+            else {
+              echo -e "\n💻 Please rerun the following command on your workstation and push your changes"
+              echo "--------------------------------------------------------------------"
+              echo "styler::style_dir('.')"
```

Comment:
Use `styler::style_file(changed_r_files)` instead. This will be a very disruptive operation to the user and the reviewer.